data/json/monsters/mutant_mammal.json

```
@@ -0,0 +1,390 @@
+[
+  {
+    "id": "mon_bear_mutant",
```

```suggestion
    "id": "mon_bear_mutant_2headed",
```
I would suggest that all of these IDs be made more precise... I have a lot of plans for other mutant animals, and when there's more variety it'll be annoying to find the right IDs if we start out vague.